### ISSUE TITLE
Request interior cabin

### ISSUE THREAD

**Issue by DEBUNKED115** [JMaverick16/Airbus_A330#80]
It's not much of an issue but rather a feature request, a cabin with seats and everything would be nice because it brings the plane more to life

**Comment by SP-NTX** [comment]
you can do it yourself :D

**Comment by DEBUNKED115** [comment]
I would but the only problem is I don't know how to feature it with an XML file, you seem to know a lot about plane files more than me :/

**Comment by sriemmanuel787** [comment]
If you have a model for the interior already, the xml shouldn't be that big of an issue.

**Comment by DEBUNKED115** [comment]
I have a copy of one from the airbus A340, it's just that it won't work if it's targeted for the A340 and not the A330

**Comment by sriemmanuel787** [comment]
No, the A340 cabin is much bigger than the A330, and it won't work without some serious editing.

**Comment by DEBUNKED115** [comment]
It has the same seat configuration, only that it's quite long
